functions/Set-DbaLogin.ps1

```diff
@@ -50,7 +50,10 @@ function Set-DbaLogin {
         Grant access to SQL Server
 
     .PARAMETER PasswordPolicyEnforced
-        Should the password policy be enforced.
+        Should the password policy be enforced (check_policy).
```

Comment:
```suggestion
        Enable the password policy on the login (check_policy = ON).
```

Saying "Should" just sounds too much like a Pester tests 🤣

Reply:
yeah, I agree